styles.css

```diff
@@ -222,8 +226,8 @@ a:hover {
   border: 1px solid #dfe1e6;
   box-sizing: border-box;
   border-radius: 16px;
-  margin-top: 20%;
-  margin-bottom: 20%;
+  margin-top: 5%;
+  margin-bottom: 5%;
 }
 
 .me {
```

Comment:
- Kindly remove empty classes.
- Try to implement in your code DRY methodology. In example:
``` 
.role {
  font-family: Poppins, sans-serif;
  font-style: normal;
  font-weight: 600;
  font-size: 13px;
  line-height: 16px;
  color: #7a869a;
  list-style-type: disc;
}

.time {
  font-family: Poppins, sans-serif;
  font-style: normal;
  font-weight: 600;
  font-size: 13px;
  line-height: 16px;
  color: #7a869a;
  list-style-type: disc;
}
```